dexs/fwx/index.ts

```diff
@@ -33,13 +33,26 @@ const fetch = (chain: Chain) => {
     const formattedDate = date.toISOString().replace(/\.(\d{3})Z$/, ".$1Z");
 
     // * call api for daily volume
-    const tradingVolumeRes = await httpPost(endpoints.tradingVolume, {
+    const tradingVolumePerpRes = await httpPost(endpoints.tradingVolume, {
       from_date: formattedDate,
       to_date: formattedDate,
       chain_id: CHAIN_ID[chain],
+      is_perp: true,
     });
-    const tradingVolume = tradingVolumeRes as IRes;
-    const dailyVolumeData = tradingVolume?.data.find(
+    const tradingVolumePerp = tradingVolumePerpRes as IRes;
+    const dailyPerpVolumeData = tradingVolumePerp?.data.find(
+      (x: IDailyData) =>
+        new Date(x.date).getTime() == new Date(formattedDate).getTime()
+    );
+
+    const tradingVolumeAphRes = await httpPost(endpoints.tradingVolume, {
+      from_date: formattedDate,
+      to_date: formattedDate,
+      chain_id: CHAIN_ID[chain],
+      is_perp: false,
```

Comment:
is_perp is spot volume ? should it include at fwx-dex ?

Reply:
nope. they are both leverage trading volume. just different features